flask_frontend/app.py

```
@@ -0,0 +1,87 @@
+            "repo_url": repo_url,
+            "token": token,
+            "repo_type": repo_type,
+        }
+        try:
+            requests.post(
+                f"{API_URL}/api/processed_projects",
+                json=params,
+                timeout=10,
+            )
+        except Exception:
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)